src/benchmark.cc

```diff
@@ -861,19 +797,17 @@ RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
             thread.join();
         }
         for (std::size_t ti = 0; ti < pool.size(); ++ti) {
-            pool[ti] = std::thread(&RunInThread, &b, iters, static_cast<int>(ti), &total);
+            pool[ti] = std::thread(&RunInThread, &b, iters,
+                                   static_cast<int>(ti + 1), &manager);
```

Comment:
I changed it so the main thread always runs as thread `0` for multi-threaded benchmarks. Previously we just slept the main thread.